### DISCUSSION TITLE
Allow .yaml extension in ISSUE_TEMPLATE/config.yml and FUNDING.yml

### DISCUSSION THREAD

**Discussion by kfiven** [community/community#13853] [Projects and Issues]
Hi, I was moving all the .yml files to .yaml in my repo and it seems `.github/ISSUE_TEMPLATE/config.yml` and `.github/FUNDING.yml` doesn't work with .yaml extension.
Since .yaml is advised format from [yaml.org](https://yaml.org/faq.html), it will be great if these two or any other (if there are) part support the new file type.

**Comment by cjbarth** [comment]
Duplicates #4569
